post-release.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# -- update POM / examples to next development version
+# -- update release properties
+# -- make sure release settings are not modified
+# -- commit
+# -- push
+
+if [[ ! "$#" -eq 2 ]]
+then
+  echo "usage: $0 NEXT_RELEASE_VERSION NEXT_DEVELOPMENT_VERSION"
+  exit 1
+fi
+
+current_release=`cat release.properties | grep "current.release=" | cut -c17-`
+next_dev=$2
+next_release=$1
```

Comment:
At this point in time `current_release` is actually the previous release, and `next_release` is the current one. How about renaming the env vars?

Reply:
Done. Also commented line where current becomes previous.